recipes/librdkafka/all/conanfile.py

```diff
@@ -73,19 +75,14 @@ def _configure_cmake(self):
         if self._cmake is not None:
             return self._cmake
         self._cmake = CMake(self)
-        self._cmake.definitions["WITHOUT_OPTIMIZATION"] = (
-            self.settings.build_type == "Debug"
-        )
+        self._cmake.definitions["WITHOUT_OPTIMIZATION"] = self.settings.build_type == "Debug"
         self._cmake.definitions["ENABLE_DEVEL"] = self.settings.build_type == "Debug"
-        self._cmake.definitions["ENABLE_REFCNT_DEBUG"] = (
-            self.settings.build_type == "Debug"
-        )
+        self._cmake.definitions["ENABLE_REFCNT_DEBUG"] = self.options.refcnt_debug
```

Comment:
```suggestion
```

Reply:
@intelligide You can mark these comments as resolved.